app/charts/shared/brush/index.tsx

```diff
@@ -28,16 +28,10 @@ export const BRUSH_HEIGHT = 3;
 
 export const BrushTime = () => {
   const ref = useRef<SVGGElement>(null);
-  const { timeRange, setTimeRange } = useInteractiveFiltersStore((d) => ({
-    timeRange: d.timeRange,
-    setTimeRange: d.setTimeRange,
-  }));
-  const { setDefaultDuration, setInstantDuration } = useTransitionStore(
-    (d) => ({
-      setDefaultDuration: d.setDefaultDuration,
-      setInstantDuration: d.setInstantDuration,
-    })
-  );
+  const timeRange = useInteractiveFiltersStore((d) => d.timeRange);
+  const setTimeRange = useInteractiveFiltersStore((d) => d.setTimeRange);
+  const setDefaultDuration = useTransitionStore((d) => d.setDefaultDuration);
+  const setInstantDuration = useTransitionStore((d) => d.setInstantDuration);
```

Comment:
Nit: 
const { timeRange, setTimeRange } =  useInteractiveFiltersStore((d) => { timeRange: d.timeRange, setTimeRange: d.setTimeRange });

Reply:
Yes, we could do it like this, but then we need the `shallow` option passed to the `useInteractiveFiltersStore` (see shallow here: https://docs.pmnd.rs/zustand/recipes/recipes), otherwise a new object is created at each render at everything re-renders 😨 With atomic picks, it does the `===` comparison by default.

I felt into this trap and spent some time finding out what's happening in other component 😅 

I can change this to destructuring with `shallow` 👍